nsv13/code/modules/mining/asteroid.dm

```diff
@@ -237,10 +237,10 @@ GLOBAL_LIST_EMPTY(asteroid_spawn_markers)		//handles mining asteroids, kind of s
 	linked.relay('nsv13/sound/effects/ship/tractor.ogg', "<span class='warning'>DANGER: Magnet has locked on to an asteroid. Vacate the asteroid cage immediately.</span>")
 	cooldown = TRUE
 	addtimer(VARSET_CALLBACK(src, cooldown, FALSE), 1 MINUTES)
-	if(prob(10)) //I hate this but it works so fuck you
+	if(prob(33)) //I hate this but it works so fuck you
```

Comment:
Not entirely sure if this is what I had to change, to increase the ruin chance.
Sorry if I am dumb maintainers.

Reply:
Yep it is, I think